tests/test_verify_base_url.py

```diff
@@ -2,70 +2,62 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import pytest
 from requests.packages.urllib3.util.retry import Retry
 
 
-@pytest.fixture(autouse=True)
-def httpserver(httpserver):
-    return httpserver
```

Comment:
For some reason, some of the tests fails when using `python 2.7` without this, seemingly, redundant fixture. Any idea why, @nicoddemus ?

Tried replacing it with `pytestmark = pytest.mark.usefixture("httpserver")` without success.

Reply:
Or maybe @davehunt knows/remembers?

Reply:
I would agree this looks redundant, but regretfully I don't recall why this was necessary.

Reply:
I just added it back, with a note about removing it when we drop 2.7 support.